### ISSUE TITLE
Tool dig speed values are too high

### ISSUE THREAD

**Issue by deftesthawk** [yanny7/ytech#17]
Most of the tools have dig speeds set to really high values, for example the bronze pickaxe has a dig speed of 32 which almost instant mines stone.

I used "Tool Stats" mod to view them in game, tools from vanilla and other mods have normal values.

**Comment by yanny7** [comment]
Please provide affected version

**Comment by deftesthawk** [comment]
Minecraft 1.20.1
YTech 0.9.2

**Comment by deftesthawk** [comment]
I don't think the basket is working correctly as it holds less than a normal minecraft inventory slot, for example it holds 16 wheat, or can only hold 1 non stackable item.

**Comment by deftesthawk** [comment]
Bottom of `Well Pulley` Item model has missing texture, I'm using an item physics mod which makes this easy to see.
![image](https://github.com/user-attachments/assets/7a6ae62c-eeca-458a-852f-60e4dc93ca81)

**Comment by yanny7** [comment]
Fixes will be included in next version

**Comment by deftesthawk** [comment]
I've been experimenting with the aqueduct system, unless I'm missing something I think the aqueduct valve needs a little tweaking. As it consumes water very fast and right clicking a water bucket straight into the aqueducts is a lot more efficient. I'm guessing this is because it was balanced around having an infinite water source, an option in the config to change the random time it takes to consume water would be nice too.

**Comment by yanny7** [comment]
True is that Valve is leftover when there wasn't Well Pulley... I have to remove it because it's useless now

**Comment by yanny7** [comment]
But I let it be... finite water source can be disabled or you can specify biomes where water is infinite... thet his block makes sense

**Comment by deftesthawk** [comment]
Yeah I think it's good to keep it in as it makes sense for people who disable finiteWaterSource

**Comment by yanny7** [comment]
Fixed in 0.10.0

**Comment by deftesthawk** [comment]
Thank you, I’m thankful for the continued support on version 1.20.1.
